nmdc_server/api.py

```diff
@@ -747,6 +755,58 @@ async def update_submission(
     return submission
 
 
+def create_github_issue(submission, user):
+    settings = Settings()
+    gh_url = str(settings.github_issue_url)
+    token = settings.github_authentication_token
+    if gh_url == None or token == None:
+        return
+
+    headers = {"Authorization": f"Bearer {token}", "Content-Type": "text/plain; charset=utf-8"}
+    studyform = submission.metadata_submission["studyForm"]
+    contextform = submission.metadata_submission["contextForm"]
+    multiomicsform = submission.metadata_submission["multiOmicsForm"]
+    pi = studyform["piName"]
+    piorcid = studyform["piOrcid"]
+    datagenerated = "Yes" if contextform["dataGenerated"] else "No"
+    omicsprocessingtypes = ", ".join(multiomicsform["omicsProcessingTypes"])
+    sampletype = ", ".join(submission.metadata_submission["templates"])
+    sampledata = submission.metadata_submission["sampleData"]
+    numsamples = 0
+    for key in sampledata:
+        numsamples = max(numsamples, len(sampledata[key]))
+
+    body_lis = [
+        f"Submitter: {user.orcid}",
+        f"Submission ID: {submission.id}",
+        f"Has data been generated: {datagenerated}",
+        f"PI name and Orcid: {pi} {piorcid}",
+        "Status: Submitted -Pending Review",
+        f"Data types: {omicsprocessingtypes}",
+        f"Sample type:{sampletype}",
+        f"Number of samples:{numsamples}",
+        "Note:",
+    ]
+    body_string = " \n ".join(body_lis)
+    payload_dict = {
+        "title": f"NMDC Submission{submission.id}",
+        "body": body_string,
+        "assignees": ["JamesTessmer"],
+        "labels": ["testing"],
```

Comment:
Just wanted to confirm that you're intentionally signing up to have all these assigned to you?

Reply:
That is a good point, that's one of the last things that needs to be done. 
I need to talk to Montana about who (if anyone) these should be assigned to and any labels they need.